### ISSUE TITLE
Update extension for new Netflix UI

### ISSUE THREAD

**Issue by Gabriel-p** [joshblum/netflix-rate-chrome-ext#14]
I'm not sure if Netflix changed its interface, but I've installed this extension for Chrome and it appears to do absolutely nothing.

**Comment by joshblum** [comment]
Yes I noticed recently that the interface changed and need some time to fix it
